pkg/virtctl/credentials/set-password/set-password.go

```diff
@@ -0,0 +1,145 @@
+package set_password
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/clientcmd"
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
+	"kubevirt.io/kubevirt/pkg/virtctl/credentials/common"
+	"kubevirt.io/kubevirt/pkg/virtctl/templates"
+)
+
+func SetPasswordCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
+	cmdFlags := &passwordCommandFlags{}
+	cmd := &cobra.Command{
+		Use:   "set-password",
+		Short: "Set password for a user",
+		Args:  templates.ExactArgs("set-password", 1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runSetPasswordCommand(clientConfig, cmdFlags, cmd, args)
+		},
+	}
+	cmdFlags.AddToCommand(cmd)
+
+	cmd.SetUsageTemplate(templates.UsageTemplate())
+	return cmd
+}
+
+const (
```

Comment:
Move this block into AddToCommand like in other places?

Reply:
done